src/managment/Reducer.tsx

```diff
@@ -0,0 +1,88 @@
+import { State, Filter } from '../types/Types';
+
+export type Action =
+  | { type: 'addTodo'; title: string }
+  | { type: 'removeTodo'; id: number }
+  | { type: 'editTitle'; id: number; newTitle: string }
```

Comment:
```suggestion
  | { type: 'editTitle'; payload: { id: number; newTitle: string } }
```
you can make the structure same for all actions which have data (payload)